src/CSSTransition.js

```diff
@@ -109,7 +109,18 @@ const propTypes = {
  * the `example-enter` CSS class and the `example-enter-active` CSS class
  * added in the next tick. This is a convention based on the `classNames` prop.
  *
- * <iframe src="https://codesandbox.io/embed/kw8z6pp9zo?autoresize=1&fontsize=12&hidenavigation=1&moduleview=1" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
+ * ---
+ *
+ * **Note**: `Transition` is platform-agnostic while `CSSTransition`
```

Comment:
this is a really great not, i've wanted to add for a while. It might be off tho in the Transition component, since that's the one folks use incorrectly all the time, so may be likely to see it.

I'd also smooth the recommendation out a bit. I think it _is_ ok and often appropriate to use Transition directly for css animations, but it's fraught if you don't know what you're doing.  It might be worth noting that this is a common problem, and if you wnat to use Transtion you need to work around it, or use CSSTransition if it fits your needs.

Reply:
Good point. I modified it a bit and added it to `Transition` page instead.